frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -200,6 +209,8 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
     } = props;
 
     const unsortedList = { ...appendItems };
+    const namespaces = new Set(_.flatten(_.map(resources, ({ data }) => data?.map(getNamespace))));
```

Comment:
What does the data look like? I'm unsure why we need `flatten` and multiple calls to `map`.